CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
   * [7269](https://github.com/stripe/stripe-android/pull/7269) iDEAL
   * [7270](https://github.com/stripe/stripe-android/pull/7270) SEPA
   * [7272](https://github.com/stripe/stripe-android/pull/7272) Sofort
+  * [7302](https://github.com/stripe/stripe-android/pull/7302) Alma
```

Comment:
Looks like this landed in the wrong release. Want to move it up to the most recent?